src/Microsoft.Health.Fhir.Core/Models/OperationOutcomeIssue.cs

```diff
@@ -26,7 +27,23 @@ public class OperationOutcomeIssue
             DetailsCodes = detailsCodes;
             DetailsText = detailsText;
             Diagnostics = diagnostics;
+            Expression = expression;
+
+            string[] location = null;
+            if (expression != null)
+            {
+                location = new string[expression.Length];
+                var i = 0;
+                foreach (var ex in expression)
+                {
+                    location[i] = ex + " // " + Resources.OperationOutcomeLocationDeprication;
```

Comment:
Thanks for moving this here inside the location. This makes my FHIR spec heart happy. ❤️‍🔥